frontend/src/components/Inventory.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
```

Comment:
can you please add the default google license to this (and all other files you created that don't have it?)